lighthouse/lighthouserc.js

```diff
@@ -0,0 +1,102 @@
+const AUDIT_URLS = {
```

Comment:
Nothing to change now but I wonder if we gain any value in testing every single page type?